tests/contrib/hooks/test_slack_webhook_hook.py

```diff
@@ -35,6 +35,7 @@ class TestSlackWebhookHook(unittest.TestCase):
         'webhook_token': 'manual_token',
         'message': 'Awesome message to put on Slack',
         'attachments': [{'fallback': 'Required plain-text summary'}],
+        'blocks': [{'type': 'section', 'text': {'type': 'mrkdwn', 'text': '*bold text*'}}],
```

Comment:
Does the version of the slack module we support (1.x, not 2.x) support this?

Reply:
It does not use the slack client module, but just HTTP post request.

Reply:
You make an excelent point :D